doc/configuration.md

```diff
@@ -55,6 +55,7 @@ _Note: all fields are optional unless otherwise marked_
     * **path** (string, required): the absolute path to the file.
     * **contents** (object): options related to the contents of the file.
       * **inline** (string): the contents of the file.
+      * **local** (string): the path to a local file, on the machine where `ct` runs.
```

Comment:
"the file on the machine where `ct` is run containing the contents of the file to be created"

cc @dgonyeo Do you have any idea for better wording?

Reply:
I think "the relative path to a file on the machine where `ct` is run. The contents of the file are included in the generated config." might be sufficient.

Reply:
This should be updated to indicate that `--files-dir` is required if it is used.